devlab/utils/clean_run.sh

```diff
@@ -34,7 +34,8 @@ python generate_load.py --clean --env DST
 python generate_load.py --clean --env DST
 ssh root@$DST \
 "mysql -u $dst_mysql_user -p$dst_mysql_password cinder -e \"delete from cinder.volume_admin_metadata;
-delete from cinder.volumes;\""
+delete from cinder.volumes;delete from cinder.reservations;delete from cinder.quota_usages;\""
```

Comment:
If you want to delete all data from table `truncate` is better.
Also you can select all table names from mysql:
`"mysql -u $dst_mysql_user -p$dst_mysql_password -Nse 'show tables' cinder | while read table; do mysql -u $dst_mysql_user -p$dst_mysql_password -e \"set foreign_key_checks=0; truncate table $table\" cinder; done"`